.storybook/addons.js

```diff
@@ -1,3 +1,21 @@
+import "core-js/es6/symbol";
```

Comment:
Weighing in a bit - Carbon vanilla/React variants maintain separate lists of "polyfills modules" so consumers know what polyfill they need to run the library on IE11. Great to have a similar thing, if the topic has not been documented.

Reply:
I'll have to double check what Angular CLI ships/recommends ... this should mirror that, or at least we should be following what they recommend.

It's probably a good idea to review our actual polyfill dependency list though, the less we have to rely on the better.